src/components/sectionList/listView/ManageListView.tsx

```diff
@@ -0,0 +1,135 @@
+import { FetchError } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import { Button, Field, NoticeBox, Transfer } from '@dhis2/ui'
+import React, { useEffect, useMemo, useRef, useState } from 'react'
+import {
+    getColumnsForSection,
+    useModelSectionHandleOrThrow,
+} from '../../../lib'
+import css from './ManageListView.module.css'
+import { useModelListView, useMutateModelListViews } from './useModelListView'
+
+interface RenderProps {
+    handleSave: () => void
+    isSaving: boolean
+}
+type ManageColumnsDialogProps = {
+    onSaved: () => void
+    children: (props: RenderProps) => React.ReactNode
+}
+
+const toPath = (propertyDescriptor: { path: string }) => propertyDescriptor.path
+
+export const ManageListView = ({
+    onSaved,
+    children,
+}: ManageColumnsDialogProps) => {
+    const section = useModelSectionHandleOrThrow()
+    // ignore updates to saved-columns while selecting
+    const isTouched = useRef(false)
+
+    const { columns: savedColumns, query } = useModelListView()
+    const [pendingSelectedColumns, setPendingSelectedColumns] = useState<
+        string[]
+    >(() => savedColumns.map(toPath))
+    const [error, setError] = useState<string | undefined>()
+    const [saveError, setSaveError] = useState<FetchError | undefined>()
```

Comment:
This will probably be updated to use a `Form` once we implement another field (eg. for selecting filters).